### PR TITLE
Pstanek/ocdm api switch 2.24 port/2

### PR DESCRIPTION
**Pull request by pawel-stanek-ml**

### CODE REVIEW

File: Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp [hunk 1]
```diff
@@ -357,44 +538,52 @@ void CDMInstanceOpenCDMSession::requestLicense(LicenseType licenseType, const At
...
+            session->generateChallenge(WTFMove(generateChallenge));
         }
         return;
     }
 
-    // FIXME: Why do we have this weirdness here? It looks like this is a way to reference count on the OpenCDM object.
-    media::OpenCdm openCDM(m_openCDM);
-    std::string sessionId = openCDM.CreateSession(m_cdmInstance->mimeType(), reinterpret_cast<const uint8_t*>(rawInitData->data()), rawInitData->size(), nullptr, 0, openCDMLicenseType(licenseType));
+    // FIXME:
+    auto rawCustomData = SharedBuffer::create();
```
Comment (pawel-stanek-ml) [inline Source/WebCore/platform/graphics/gstreamer/eme/CDMOpenCDM.cpp:510]: Be aware of this added FIXME needed due to the fact customData isn't there yet

**Comment by pawel-stanek-ml** [comment]
@charlie-ht Any chances of this getting in?

**Comment by charlie-ht** [comment]
Unfortunately I'm not able to build metrological's buildroot with 2.24 and the framework revisions I was last given. I'm working on getting this built in an upstream buildroot instead for the time being. Once that is working I'll be able to validate the changes here.

**Comment by pawel-stanek-ml** [comment]
Ok. Thanks

**Comment by pawel-stanek-ml** [comment]
The code has significantly changed so closing this.

**Pull request closed without merging**